lib/helpers/broadcast.js

```diff
@@ -35,23 +35,28 @@ async function fetch(query = {}) {
  */
 function fetchById(broadcastId) {
   return contentful.fetchByContentfulId(broadcastId)
-    .then(contentfulEntry => module.exports.parseBroadcastFromContentfulEntry(contentfulEntry));
+    .then(contentfulEntry => module.exports.parseBroadcastFromContentfulEntry(contentfulEntry))
+    .then(broadcast => helpers.cache.broadcasts.set(broadcastId, broadcast));
 }
 
 /**
- * @param {String} topicId
+ * @param {String} broadcastId
+ * @param {Boolean} resetCache
  * @return {Promise}
  */
-function getById(broadcastId) {
+function getById(broadcastId, resetCache = false) {
+  if (resetCache) {
```

Comment:
Should also check strict equality here like we do with [topics](https://github.com/DoSomething/gambit-campaigns/pull/1064/files#diff-fe5513ac2c68bbe898ecb4288a671ac4R67)?